c7n/filters/core.py

```diff
@@ -220,7 +221,7 @@ class ValueFilter(Filter):
             'type': {'enum': ['value']},
             'key': {'type': 'string'},
             'value_type': {'enum': [
-                'age', 'integer', 'expiration', 'normalize', 'size']},
+                'age', 'integer', 'expiration', 'normalize', 'size', 'cidr_size']},
```

Comment:
so the ipaddress network classes support rich comparisons, the operator 'op' config setting allows changing that directly ( > = <, in, not-in, etc), which is nice, ie. directly mirror the operators from a value that has rich comparisons, ie. changing to just a value_type: cidr, you can do most of the operators that way.